### DISCUSSION TITLE
why so much RAM?

### DISCUSSION THREAD

**Discussion by ghost** [WerWolv/ImHex#1022] [Feedback]
why does this program require 300 MB of ram on idle? the other hex editor I know, which unfortunately is closed source, only consumes 3 MB of ram on idle. all that memory consumption would be understandable for an eletron program, but this is a C++ one.

**Comment by WerWolv** [comment]
Hey, most of it is because we load all unicode characters from unifont so they can be displayed in the interface. If you don't want that, you can turn off the "Load all unicode characters" option in the settings which reduces the RAM usage to about 30MB. 
The rest is mostly resources like images and text that takes quite a bit of memory. 

I think that's mostly an acceptable level for a Desktop application in 2023
